<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveChangeAuto</class>
 <widget class="QDialog" name="RemoveChangeAuto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="Combo_DelChangeAutoRegs">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="L_ChangeAutoMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Корекция по автомобил</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Back">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_ChangeClientName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Име на 
 клиента</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="Combo_ClientName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Record">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Запис</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_ChangeAutoRegs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Регистрационнен 
 номер</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>130</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="L_Change_Type">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Купе</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QComboBox" name="m_ComboFuel">
      <item>
       <property name="text">
        <string>Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Бензин</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Дизел</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Газ/Бензин</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Метан/Бензин</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Хибрид</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Електрически</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="L_Change_RegNumber">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Регистрационен 
 номер</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="L_Change_Marka">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Марка</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QComboBox" name="m_ComboModel"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="L_Change_Model">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Модел</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="L_Change_Year">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Година</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="L_Change_VIN">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>РАМА / VIN /</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QComboBox" name="m_ComboType">
      <item>
       <property name="text">
        <string>Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Седан</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Хечбек</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Комби</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Купе</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кабрио</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Джип</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Пикап</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ван</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QComboBox" name="m_ComboYear">
      <property name="currentText">
       <string>Select</string>
      </property>
      <item>
       <property name="text">
        <string>Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2018</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2017</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2016</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2015</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2014</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2013</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2012</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2011</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2010</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2009</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2008</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2007</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2006</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2005</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2004</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2003</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2002</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2001</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1999</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1998</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1997</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1996</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1995</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1994</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1993</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1992</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1991</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1990</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1989</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1988</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1987</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1986</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1985</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLineEdit" name="LText_DelChangeVIN"/>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="L_ChangeAutoInfo">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Данни за автомобила </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QComboBox" name="m_ComboMarka"/>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="LText_DelChangeRegNumber"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="L_Change_Fuel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Гориво</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="L_Change_Millage">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Пробег ( км )</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QLineEdit" name="LText_AutoMillage"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Button_DeleteAuto">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Изтрий 
 автомобил</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Add">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Добави</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LText_ClientName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="L_LineClientName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Име на клиента</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L_UnkownClients">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Изброените автомобили не принадлежат към никой клиент</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
